### ISSUE TITLE
List item link URLs when section titles start with an emoji

### ISSUE THREAD

**Issue by MalloryWittwer** [sindresorhus/awesome-lint#222]
I'm working on the list [awesome-scientific-image-analysis](https://github.com/EPFL-Center-for-Imaging/awesome-scientific-image-analysis).

The output of the command

```sh
npx awesome-lint https://github.com/EPFL-Center-for-Imaging/awesome-scientific-image-analysis
```

suggests that the list items used in the *Contents* are invalid:

```
✖  12:3  Invalid list item link URL  remark-lint:awesome-list-item
...
```
However, I believe the syntax for creating these links is actually correct, and the links do work fine in the GitHub readme. Am I missing something?

For reference, here's how it's done in the README:

```markdown
## 📌 Contents

- [🔖 Getting started](#-getting-started)
- [🧑‍🤝‍🧑 Communities](#-communities)
- [📚 Learning resources](#-learning-resources)
...
```

Could it be that the linter is confused about the use of the dash symbol to link to a section title starting with an emoji?

Thank you for your help!

**Comment by MalloryWittwer** [comment]
Solved it! Actually, the issue was not caused by the list item link URLs, but by the 📌 emoji in front of "Contents". The linting error disappeared as soon as I removed that emoji. That makes sense!
I will close this issue. Thank you for this excellent project.
